sites/fast-website/src/app/css/style.css

```diff
@@ -62,13 +63,46 @@ li {
   display: none;
 }
 
+fast-divider {
+  border: none;
+  display: flex;
+  flex-direction: column;
+  height: 58px;
+  justify-content: space-between;
+  margin: calc(var(--design-unit) * 5px) calc(50% - 1.5px);
+  width: 3px;
+}
+
+fast-divider::before,
+fast-divider::after {
+  background-color: var(--accent-fill-rest);
+  content: '';
+  width: 3px;
+}
+
+fast-divider::before {
+  border-radius: 50%;
+  height: 3px;
+}
+
+fast-divider::after {
+  border-radius: calc(var(--corner-radius) * 1px);
+  height: calc(var(--type-ramp-plus-5-font-size) + var(--design-unit) * 1px);
+}
+
 .section {
-  grid-column: 2;
+  --card-width: 400px;
```

Comment:
Will this card width cause us issues on smaller mobile screens? I believe that before the grid in the container component was dealing with the sizing. I am also noticing that some of the cards are being cut off on screen resize that I believe might be related to this.

Reply:
This shouldn't be in here. I'll remove it.